### DISCUSSION TITLE
Version 1.8.2 Should have been a Major Version Bump

### DISCUSSION THREAD

**Discussion by epoctic** [danestves/remix-auth-auth0#101] [General]
Hello,

I'd like to make the case that the version 1.8.0 (and subsequent patch 1.8.2) should have been a major version bump because it introduces a dependency resolution breaking change through the major version bump from remix 1.x to 2.x. I understand the idea that technically no breaking changes were made in this library alone, but the change in the library's dependencies is so great that I think it warrants the bump anyway.

This was first discovered when a number of our image building pipelines began to fail yesterday afternoon (EST), after which I began troubleshooting locally and found the problem to be easily reproducible.

For reference, the failed dependency resolution was confirmed on the following node/npm versions combinations:
- node18.18.2 npm9.8.1
- node18.18.0 npm10.2.1

I have a stripped down version of the package.json for you to try here:
```
{
  "name": "auth0test",
  "description": "UI for auth0test",
  "dependencies": {
    "@remix-run/dev": "1.6.8",
    "@remix-run/express": "1.6.8",
    "@remix-run/node": "1.6.1",
    "@remix-run/react": "1.6.8",
    "@remix-run/serve": "1.6.8",
    "@rollbar/react": "^0.11.0",
    "auth0-js": "^9.19.0",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-select": "^5.2.2",
    "remix-auth": "^3.2.1",
    "remix-auth-auth0": "1.4.0"
  },
  "devDependencies": {
    "@remix-run/eslint-config": "1.6.8",
    "@types/aspnet-identity-pw": "^1.0.30",
    "@types/auth0-js": "^9.14.7",
    "@types/react": "^17.0.24",
    "@types/react-dom": "^17.0.9",
    "@vitejs/plugin-react": "^1.3.1",
    "typescript": "^4.7.4"
  },
  "engines": {
    "node": ">=16"
  }
}
```

**Comment by danestves** [comment]
Since version `1.8.0` we are requesting to have all `@remix-run/*` packages on `v2`, we require this because the underlying package `remix-auth` will require that too.

**Reply by epoctic** [reply]
Do you know when remix-auth said they would exclusively be supporting remixv2? Currently they support both: https://github.com/sergiodxa/remix-auth/pull/243/files

I wasn't able to find anything in their discussions page that had any relevant information.
